### ISSUE TITLE
NPC: Blastmaster Emi Shortfuse

### ISSUE THREAD

**Issue by underlord-oberon** [plantain-gnocchis/wow-voiceover-tbc-wotlk#73]
Add-on version: AI_VoiceOver_v1.4.5
Data version: AI_VoiceOverData_VanillaExtra_v1.4

Gnomeregan / The Hall of Gears / NPC: Blastmaster Emi Shortfuse
Link: https://www.wowhead.com/wotlk/npc=7998/blastmaster-emi-shortfuse

Greeting: Despite not been a formal quest, this NPC starts an event inside the dungeon which can be considered a quest. And offers a [gossip or greeting text](https://wowpedia.fandom.com/wiki/Blastmaster_Emi_Shortfuse) explaining the event.

**Comment by carevass** [comment]
This should be fixed with new audio in AI_VoiceOverData_VanillaExtra_v1.5.
